xilem/examples/http_cats/main.rs

```diff
@@ -170,21 +179,26 @@ impl Status {
 
     fn details_view(&mut self) -> impl WidgetView<HttpCats> {
         let image = match &self.image {
-            ImageState::NotRequested => {
-                OneOf3::A(prose("Failed to start fetching image. This is a bug!"))
-            }
-            ImageState::Pending => OneOf3::B(sized_box(spinner())),
+            ImageState::NotRequested => OneOf3::A(
+                prose("Failed to start fetching image. This is a bug!")
+                    .alignment(TextAlignment::Middle),
+            ),
+            ImageState::Pending => OneOf3::B(sized_box(spinner()).width(80.).height(80.)),
+            // TODO: Alt text?
             ImageState::Available(image_data) => OneOf3::C(image(image_data.clone())),
```

Comment:
Now, once the image is loaded, it takes up a lot of vertical space. Is this avoidable? At the default size of the window, it pushes things far enough down to need to scroll?

Reply:
I'm trying to debug this - Masonry's image widget has strange behaviour here.

Reply:
The latest commit fixes it. Perfect!